### ISSUE TITLE
ssllyze on Windows 10 and Microsoft Visual Studio 2017 Community

### ISSUE THREAD

**Issue by sw42x** [nabla-c0d3/sslyze#240]
... problems with adding the package to the python-environment....

this is the log of installing .... 
```

----- "sslyze" wird installiert -----
Collecting sslyze
  Using cached SSLyze-1.1.2.zip
Collecting nassl<0.17.0,>=0.16.0 (from sslyze)
  Using cached nassl-0.16.3.zip
Collecting cryptography==1.9 (from sslyze)
  Using cached cryptography-1.9-cp36-cp36m-win_amd64.whl
Collecting tls-parser==1.0.0 (from sslyze)
  Using cached tls_parser-1.0.0.zip
Collecting cffi>=1.7 (from cryptography==1.9->sslyze)
  Using cached cffi-1.10.0-cp36-cp36m-win_amd64.whl
Collecting six>=1.4.1 (from cryptography==1.9->sslyze)
  Using cached six-1.10.0-py2.py3-none-any.whl
Collecting asn1crypto>=0.21.0 (from cryptography==1.9->sslyze)
  Using cached asn1crypto-0.22.0-py2.py3-none-any.whl
Collecting idna>=2.1 (from cryptography==1.9->sslyze)
  Using cached idna-2.6-py2.py3-none-any.whl
Collecting pycparser (from cffi>=1.7->cryptography==1.9->sslyze)
  Using cached pycparser-2.18.tar.gz
Installing collected packages: nassl, pycparser, cffi, six, asn1crypto, idna, cryptography, tls-parser, sslyze
  Running setup.py install for nassl: started
    Running setup.py install for nassl: finished with status 'error'
    Complete output from command "C:\Program Files\Python36\python.exe" -u -c "import setuptools, tokenize;__file__='C:\\Users\\Wilfried\\AppData\\Local\\Temp\\pip-build-1mmrtorx\\nassl\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\Wilfried\AppData\Local\Temp\pip-p59eba5x-record\install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.6
    creating build\lib.win-amd64-3.6\nassl
    copying nassl\__init__.py -> build\lib.win-amd64-3.6\nassl
    copying nassl\ssl_client.py -> build\lib.win-amd64-3.6\nassl
    copying nassl\debug_ssl_client.py -> build\lib.win-amd64-3.6\nassl
    copying nassl\ocsp_response.py -> build\lib.win-amd64-3.6\nassl
    running build_ext
    building 'nassl._nassl' extension
    creating build\temp.win-amd64-3.6
    creating build\temp.win-amd64-3.6\Release
    creating build\temp.win-amd64-3.6\Release\nassl
    creating build\temp.win-amd64-3.6\Release\nassl\_nassl
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.11.25503\bin\HostX64\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MT -Ibin\openssl\include "-IC:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\include" "-IC:\Program Files\Python36\include" "-IC:\Program Files\Python36\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.11.25503\Include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" /Tcnassl/_nassl/nassl.c /Fobuild\temp.win-amd64-3.6\Release\nassl/_nassl/nassl.obj
    nassl.c
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.11.25503\bin\HostX64\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MT -Ibin\openssl\include "-IC:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\include" "-IC:\Program Files\Python36\include" "-IC:\Program Files\Python36\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.11.25503\Include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" /Tcnassl/_nassl/nassl_SSL_CTX.c /Fobuild\temp.win-amd64-3.6\Release\nassl/_nassl/nassl_SSL_CTX.obj
    nassl_SSL_CTX.c
    C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.11.25503\bin\HostX64\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MT -Ibin\openssl\include "-IC:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\include" "-IC:\Program Files\Python36\include" "-IC:\Program Files\Python36\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.11.25503\Include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" /Tcnassl/_nassl/nassl_SSL.c /Fobuild\temp.win-amd64-3.6\Release\nassl/_nassl/nassl_SSL.obj
    nassl_SSL.c
    C:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\um\winsock2.h(882): fatal error C1083: Datei (Include) kann nicht geöffnet werden: "qos.h": No such file or directory
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Tools\\MSVC\\14.11.25503\\bin\\HostX64\\x64\\cl.exe' failed with exit status 2
    
    ----------------------------------------
Command ""C:\Program Files\Python36\python.exe" -u -c "import setuptools, tokenize;__file__='C:\\Users\\Wilfried\\AppData\\Local\\Temp\\pip-build-1mmrtorx\\nassl\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\Wilfried\AppData\Local\Temp\pip-p59eba5x-record\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\Wilfried\AppData\Local\Temp\pip-build-1mmrtorx\nassl\
----- Fehler beim Installieren von "sslyze" -----
```

what's to do ?

**Comment by nabla-c0d3** [comment]
Python 3.6 is not supported yet on Windows. It will be in the next release.

**Comment by nottheend** [comment]
While waiting sslyze can be run on Windows with a python2.7 enviroment, created by _Anaconda_
Issue #210 is also windows-concerned

**Comment by nabla-c0d3** [comment]
The master branch of SSLyze should already support Python 3.6 on Windows (but I haven't done any testing yet :p )

**Comment by nabla-c0d3** [comment]
This is fixed in 1.1.3 https://github.com/nabla-c0d3/sslyze/releases/tag/1.1.3
